yesod-test/yesod-test.cabal

```diff
@@ -14,29 +14,30 @@ description:     API docs and the README are available at <http://www.stackage.o
 extra-source-files: README.md, LICENSE, test/main.hs, ChangeLog.md
 
 library
-    build-depends:   base                      >= 4.3      && < 5
```

Comment:
Generally I keep such changes separate, but not a big deal. I'll merge once Travis goes green.